lms/djangoapps/instructor/enrollment.py

```diff
@@ -244,6 +245,15 @@ def reset_student_attempts(course_id, student, module_state_key, requesting_user
                     requesting_user_id=requesting_user_id
                 )
                 submission_cleared = True
+                SCORE_CHANGED.send(
```

Comment:
Correct me if I'm wrong here, and feel free to flag it as out of scope for this change, but do we even need `points_earned` and `points_possible` to be sent with the `SCORE_CHANGED` signal? I went digging into the handler and it looks like we're not even using those values any longer. https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/signals/handlers.py#L85